### PR TITLE
docs: Artillery.io executor

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/executor-artillery.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 600
+      arrivalRate: 50
+      name: Sustained load
+```
+
+Now we can create a new Testkube based on the saved Artillery test deifnition.
```
Comment (vsukhin) [inline docs/executor-artillery.md:43]: new Testkube test

File: docs/executor-artillery.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## **Create a New Testkube Test Script**
+
```
Comment (vsukhin) [inline docs/executor-artillery.md:46]: Script?

File: docs/executor-artillery.md (hunk 1 above)
Comment (vsukhin) [reply docs/executor-artillery.md:43]: definition

**Review by vsukhin — APPROVED** [review]
some typos

File: docs/executor-artillery.md (hunk 1 above)
Comment (nicufk) [reply docs/executor-artillery.md:43]: We can use Add a suggestion option to be more efficient 😄

Comment (nicufk) [inline docs/executor-artillery.md:43]:
```suggestion
Now we can create a new Testkube test based on the saved Artillery test definition.
```

Comment (nicufk) [reply docs/executor-artillery.md:43]: @vsukhin ^^^

File: docs/executor-artillery.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
+# Artillery.io performance tests
```
Comment (jfermi) [inline docs/executor-artillery.md:1]: Artillery.io Performance Tests

File: docs/executor-artillery.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+curl http://testkube-api-server:8088/health
+```
+
+Output:
+
+```sh
+200 OK
+```
+
+## Create a test manifest
```
Comment (jfermi) [inline docs/executor-artillery.md:22]: Create a Test Manifest

File: docs/executor-artillery.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+The Artillery tests are defined in declarative manner, as YAML files.  
+Test should warm up our service a little bit first, next we can hit a little harder. 
+
+Let's save our test into `test.yaml` file with content below:
```
Comment (jfermi) [inline docs/executor-artillery.md:27]: Let's save our test into `test.yaml` file with the content below:

File: docs/executor-artillery.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+      arrivalRate: 50
+      name: Sustained load
+scenarios:
+  - name: "Check health endpoint"
+    flow:
+      - get:
+          url: "/health"
+```
+
+Our test is ready, but how to run it in Kubernetes cluster? Testkube will help you with that!
```
Comment (jfermi) [inline docs/executor-artillery.md:50]: Our test is ready, but how do we run it in a Kubernetes cluster? Testkube will help you with that!

File: docs/executor-artillery.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+  p99: ......................................................................... 12.6
+Log file: /tmp/test-report.json
+
+Status Test execution completed with success 🥇
+
+```
+
+## Summary
+
+With Artillery executor you can now run your tests in Kubernetes with ease.
```
Comment (jfermi) [inline docs/executor-artillery.md:169]: With the Artillery executor you can now run your tests in Kubernetes with ease.

**Review by nicufk — APPROVED** [review]
LGTM

**Review by jfermi — APPROVED** [review]

**Pull request merged**